addons/website/models/website.py

```diff
@@ -467,7 +467,7 @@ def _force(self):
 
     def _force_website(self, website_id):
         if request:
-            request.session['force_website_id'] = website_id and int(website_id)
+            request.session['force_website_id'] = website_id and str(website_id).isdigit() and int(website_id)
```

Comment:
@JKE-be This will cover cases where website_id is a string or an integer but it will crash later on if the website_id does not exists..
Need to cover that case too?